app-frontend/src/app/components/common/listItem/index.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
```

Comment:
Didn't you say we shouldn't do this for bundling reasons?

Reply:
nope
we should avoid using `/* globals {any library} */` wherever possible
`import _` is a) easier to find using your favorite grep-like tool and b) recognized and processed correctly by webpack, so you can use one of the several lodash optimization plugins whenever we get around to working on bundle optimization again.

Reply:
we don't currently do any bundle optimization on it

Reply:
Ah I see -- I must have remembered wrong